test/e2e/branding_test.go

```diff
@@ -0,0 +1,74 @@
+package e2e
+
+import (
+	"testing"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	operatorsv1 "github.com/openshift/api/operator/v1"
+	consoleapi "github.com/openshift/console-operator/pkg/api"
+	"github.com/openshift/console-operator/pkg/testframework"
+)
+
+// Test the customization features of the config map
+
+// the simpler test first would be branding set via the operator config
+func TestOperatorConfigBranding(t *testing.T) {
+	expectedBrand := operatorsv1.BrandDedicated
+	client := testframework.MustNewClientset(t, nil)
+	originalBrand := testframework.GetOperaterConfigBrand(t, client)
+
+	// helper to update the operator config
+	testframework.SetOperaterConfigBrand(t, client, expectedBrand)
+
+	// now check the console-config in openshift-console & verify it has brand set
+	gotBrand := testframework.GetOperaterConfigBrand(t, client)
```

Comment:
Your comment says `check console-config` brand but you are calling `GetOperatorConfigBrand()`?

I don't think this works.